ui/page/privacy/setup_mixer_accounts_page.go

```diff
@@ -62,162 +52,150 @@ func NewSetupMixerAccountsPage(l *load.Load, dcrWallet *dcr.Asset) *SetupMixerAc
+					label := pg.Theme.H6(values.String(values.StrSetUpStakeShuffleAutoOrManualA))
+					return layout.Inset{
+						Left: values.MarginPadding24,
+					}.Layout(gtx, label.Layout)
+				}),
+				layout.Rigid(func(gtx C) D {
+					label := pg.Theme.Body1(values.String(values.StrSetUpStakeShuffleAutoOrManualB))
+					return layout.Inset{
+						Left: values.MarginPadding24,
+					}.Layout(gtx, label.Layout)
+				}),
+				layout.Rigid(func(gtx C) D {
+					// TODO: Find a way to make Mixed and Unmixed bold while keeping to the theme.
+					label := pg.Theme.Body1(values.String(values.StrSetUpStakeShuffleAutoOrManualC))
+					return layout.Inset{
+						Top:  values.MarginPadding10,
+						Left: values.MarginPadding24,
+					}.Layout(gtx, label.Layout)
+				}),
+				layout.Rigid(func(gtx C) D {
+					label := pg.Theme.Body1(values.String(values.StrSetUpStakeShuffleAutoOrManualD))
+					return layout.Inset{
+						Top:  values.MarginPadding10,
+						Left: values.MarginPadding24,
+					}.Layout(gtx, label.Layout)
+				}),
+				layout.Rigid(func(gtx C) D {
+					return layout.Spacer{Height: values.MarginPadding80}.Layout(gtx)
+				}),
+				layout.Rigid(func(gtx C) D {
+					line := pg.Theme.Separator()
+					return layout.Inset{
+						Top:   values.MarginPadding10,
+						Left:  values.MarginPadding24,
+						Right: values.MarginPadding24,
+					}.Layout(gtx, line.Layout)
+				}),
+				layout.Rigid(func(gtx C) D {
+					return pg.autoSetupClickable.Layout(gtx, pg.autoSetupLayout)
+				}),
+				layout.Rigid(func(gtx C) D {
+					return pg.manualSetupClickable.Layout(gtx, pg.manualSetupLayout)
+				}),
+				layout.Rigid(func(gtx C) D {
+					return layout.Spacer{Height: values.MarginPadding10}.Layout(gtx)
+				}),
+			)
+		})
+	})
 }
 
 func (pg *SetupMixerAccountsPage) autoSetupLayout(gtx C) D {
 	gtx.Constraints.Min.X = gtx.Constraints.Max.X
-	return layout.UniformInset(values.MarginPadding16).Layout(gtx, func(gtx C) D {
+	gtx.Constraints.Min.Y = int(values.MarginPadding70)
+	gtx.Constraints.Max.Y = gtx.Constraints.Min.Y
+	return layout.Inset{
+		Top:    values.MarginPadding10,
+		Bottom: values.MarginPadding10,
+	}.Layout(gtx, func(gtx C) D {
 		return layout.Flex{Spacing: layout.SpaceBetween}.Layout(gtx,
-			layout.Rigid(func(gtx C) D {
-				return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
+			layout.Flexed(5, func(gtx C) D {
+				return layout.Flex{Axis: layout.Vertical, Alignment: layout.Start}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						label := pg.Theme.H6(values.String(values.StrSetUpStakeShuffleAutoTitle))
+						return layout.Inset{
+							Left: values.MarginPadding24,
+						}.Layout(gtx, label.Layout)
+					}),
 					layout.Rigid(func(gtx C) D {
-						return pg.autoSetupIcon.Layout(gtx, values.MarginPadding20)
+						label := pg.Theme.Body1(values.String(values.StrSetUpStakeShuffleAutoDesc))
+						return layout.Inset{
+							Left: values.MarginPadding24,
+						}.Layout(gtx, label.Layout)
```

Comment:
can you inset all of it before flex, `Left: values.MarginPadding24` duplicate

Reply:
Is now inset to 24 with the containing layout.